components/help-and-support/SearchTopicsSection.js

```diff
@@ -14,10 +23,97 @@ const SearchInput = styled(StyledInput)`
   border: none;
 `;
 
+const SearchResultPopup = styled(StyledCard)`
+  border: 1px solid rgba(50, 51, 52, 0.05);
+  box-shadow: 0px 18px 40px rgba(0, 0, 0, 0.0059351), 0px 7.56604px 12.0812px rgba(0, 0, 0, 0.0111057),
+    0px 4.10431px 6.12521px rgba(0, 0, 0, 0.0160729), 0px 2.27585px 3.41503px rgba(0, 0, 0, 0.0209117),
+    0px 1.1708px 1.84491px rgba(0, 0, 0, 0.0256307), 0px 0.463169px 0.792047px rgba(0, 0, 0, 0.03);
+  border-radius: 16px;
+  padding: 20px;
+  z-index: 1;
+`;
+
+const SectionCard = styled(StyledCard)`
+  border: none;
+
+  &:hover {
+    background: ${themeGet('colors.blue.600')};
+
+    p {
+      color: white;
+    }
+  }
+`;
+
+function getAllSections(items) {
+  return items.reduce((acc, item) => {
+    return [...acc, ...item.sections];
+  }, []);
+}
+
+const DOCS_BASE_URL = 'https://docs.opencollective.com';
+
+const REACT_POPPER_MODIFIERS = [
+  {
+    name: 'offset',
+    options: {
+      offset: [0, 10],
+    },
+  },
+];
+
+const LoadingSearchResults = () => {
+  const placeholderNum = 4;
+  return Array.from({ length: placeholderNum }, (_, i) => (
+    <React.Fragment key={i}>
+      <LoadingPlaceholder height="62px" borderRadius="4px" />
+      {i !== placeholderNum - 1 && <StyledHr my="3px" width="100%" borderColor="rgba(50, 51, 52, 0.1)" />}
+    </React.Fragment>
+  ));
+};
+
 const SearchTopics = () => {
+  const innerRef = React.useRef();
+  const [refElement, setRefElement] = React.useState(null);
+  const [popperElement, setPopperElement] = React.useState(null);
+  const [showSearchResults, setShowSearchResults] = React.useState(null);
+  const [searchResults, setSearchResults] = React.useState([]);
+  const [searchQuery, setSearchQuery] = React.useState('');
+  const [isLoading, setIsLoading] = React.useState(false);
+  const { styles, attributes } = usePopper(refElement, popperElement, {
+    placement: 'bottom',
+    modifiers: REACT_POPPER_MODIFIERS,
+  });
+
+  useGlobalBlur(innerRef, outside => {
+    if (outside && showSearchResults) {
+      setShowSearchResults(false);
+    }
+  });
+  const sections = React.useMemo(() => getAllSections(searchResults), [searchResults]);
+
+  const search = async query => {
+    if (!query) {
+      setSearchResults([]);
+      return;
+    }
+
+    try {
+      setIsLoading(true);
+      const results = await searchDocs(query);
+      setSearchResults(results.items);
+    } catch (error) {
+      console.error('error', error);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const debouncedSearch = React.useCallback(debounce(search, 200), []);
```

Comment:
@Betree is adequate debounce for you?

Was doing a test and thought it could be too short or misconfigured?

<img width="1831" alt="Screenshot 2023-03-28 at 13 48 57" src="https://user-images.githubusercontent.com/806/228227478-9d612e97-d773-4474-8d7b-2b8f63edd065.png">